ReleaseNotes.md

```diff
@@ -5,3 +5,4 @@
 - Updated to Point Android SDK v15.4.2
 - Updated to Point iOS SDK v15.6.4
 - Fix `"new NativeEventEmitter()" was called with a non-null argument without the required "some" method` warnings which appear with using react-native versions `0.65.0` or higher.
```

Comment:
Some of the release notes I believe are outdated 🤔 
Also, can we take this chance to update to the latest Android/iOS SDKs?

Reply:
Sure